aggregator/src/datastore.rs

```diff
@@ -2885,6 +3070,155 @@ ORDER BY id DESC
                 .try_into()?,
         ))
     }
+
+    /// Deletes old client reports for a given task, that is, client reports whose timestamp is
+    /// older than a given timestamp which are not included in any report aggregations.
+    #[tracing::instrument(skip(self), err)]
+    pub async fn delete_old_client_reports(
```

Comment:
The schema has `report_expiry_age`, so I think we should be consistent:
```suggestion
    pub async fn delete_expired_client_reports(
```